corehq/apps/hqadmin/views/users.py

```diff
@@ -480,29 +480,27 @@ def get(self, request, *args, **kwargs):
 
     def form_valid(self, form):
         from django_otp import devices_for_user
-        change_messages = {}
+        devices_reset = False
 
         username = form.cleaned_data['username']
         user = User.objects.get(username__iexact=username)
         for device in devices_for_user(user):
             device.delete()
-        change_messages.update(UserChangeMessage.registered_devices_reset())
+            devices_reset = True
```

Comment:
do we need to record this? It isn't an option action so we could assume that if 2fa was disabled then it was done right? I don't see any additional value in recording it.

Reply:
https://github.com/dimagi/commcare-hq/pull/30253/commits/48cb8fad73224f5f978040fb32736e5a3d99301d